ethcore/src/block.rs

```diff
@@ -599,8 +617,8 @@ impl SealedBlock {
 
 impl Drain for SealedBlock {
 	/// Drop this object and return the underlieing database.
```

Comment:
"…**underlying** database"

Also: is it true that it's a "database" being returned with this change?

Reply:
it is! good catch!